x-pack/plugins/cases/server/client/configure/utils.ts

```diff
@@ -75,7 +90,7 @@ export const formatFields = (theData: unknown, theType: string): ConnectorField[
   }
 };
 
-const getPreferredFields = (theType: string) => {
+const getPreferredFields = (theType: string, swimlaneMappings?: SwimlaneMappings) => {
```

Comment:
Why the need of the `swimlaneMappings` ?

Reply:
This make the function very specific to `swimlane` but its purpose is more generic. Passing the connector could be a better approach.